### ISSUE TITLE
Add more prefixes for Robot#listen

### ISSUE THREAD

**Issue by ianagbip1oti** [princesslana/ErisCasper.java#93]
By default it should listen to
* `+` blah blah
* `BotName` blah blah
* `@BotMention` blah blah

This depends upon #92

**Comment by WangzoR** [comment]
Should we consider making these optionally parametised/configurable?
ie. user of lib can dictate their own preferred prefix (~, ++, +, etc etc)optional disabling of `BotName` and optional disabling of `@BotMention`

if no overrides provided, default behavior is as you have set out above

**Comment by ianagbip1oti** [comment]
Configuration I consider a nice to have, and not essential.

If you want to include configuration then go for it, otherwise I'm happy with just the defaults for now.
